core/dbt/include/global_project/macros/materializations/common/merge.sql

```diff
@@ -37,7 +38,7 @@
 
     {% if unique_key %}
     when matched then update set
-        {% for column in dest_columns -%}
+        {% for column_name in update_columns -%}
             {{ adapter.quote(column.name) }} = DBT_INTERNAL_SOURCE.{{ adapter.quote(column.name) }}
```

Comment:
Now that the loop item (`column_name`) is just a string, rather than a `Column` object, you'll need to reference it as `column_name` (string) instead of `column.name` (value from object):
```suggestion
            {{ adapter.quote(column_name) }} = DBT_INTERNAL_SOURCE.{{ adapter.quote(column_name) }}
```
I think this should resolve the error in CI:
```
Database Error in model model (models/model.sql)
  Unrecognized name: `<property object at 0x7efc88e6ebd8>` at [30:9]
  compiled SQL at target/run/test/models/model.sql
```

Reply:
Got it, thanks! Yeah I got the test suite running with Docker locally but couldn't make sense of the errors.